test/framework/sandbox/easybuild/easyblocks/generic/perlbundle.py

```diff
@@ -0,0 +1,44 @@
+##
+# Copyright 2009-2025 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+Dummy easyblock for Makecp.
+
+@author: Miguel Dias Costa (National University of Singapore)
+"""
+from easybuild.framework.easyblock import EasyBlock
+from easybuild.framework.easyconfig import CUSTOM
+
+
+class PerlBundle(EasyBlock):
+    """Dummy support for bundle of modules."""
+
+    @staticmethod
+    def extra_options(extra_vars=None):
+        if extra_vars is None:
+            extra_vars = {}
+        extra_vars.update({
+            'components': [(), "List of components to install: tuples w/ name, version and easyblock to use", CUSTOM],
```

Comment:
It's a bit confusing to have this here, add it in the fake `Bundle` easyblock, and derive this easyblock from `Bundle`?

Reply:
I copied the fake python bundle easyblock and that has this in. However, it does not look to be needed at all.